data-prepper-plugins/geoip-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/geoip/provider/LocationData.java

```diff
@@ -0,0 +1,26 @@
+package com.amazon.dataprepper.plugins.prepper.geoip.provider;
+
+
+// This is a first pass of LocationData, simply to make sure lookup correctly passes through data.
+//TODO Find the best way to attach the location data to a document, and modify this class to allow that.
+
+public class LocationData {
+    public String countryName;
+    public String subdivisionName;
+    public String cityName;
+
+    public LocationData(String country, String subdivision, String city){
+        countryName = country;
+        subdivisionName = subdivision;
+        cityName = city;
+    }
+
+    @Override
+    public String toString() {
+        return "LocationData{" +
```

Comment:
Instead of hardcode the json format like string, can we utilize jackson ObjectMapper?